Update installation instructions (#313)
TomographicImaging/iDVC: docs/source/installation.rst

Installation
************

> Binary installation of iDVC can be achieved with conda. Currently we only support Windows, although a Linux version is also available.

Reviewer: ```suggestion
Binary installation of iDVC can be achieved with conda. Currently we only tested the app on Windows.
```

1.	Install miniconda<https://docs.conda.io/en/latest/miniconda.html>
2.	Open an anaconda prompt (miniconda) and type….

> 3.	``conda create --name dvc-core ccpi-dvc -c ccpi -c conda-forge -c defaults --override-channels``

Reviewer: ```suggestion
3.	``conda create --name dvc-core ccpi-dvc -c ccpi -c conda-forge ``
```
